Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: all deps static clean client-lint client-test client-sync backend frontend shell lint
 
 # If you can use Docker without being root, you can `make SUDO= <target>`
-SUDO=$(shell (echo "$$DOCKER_HOST" | grep "tcp://" >/dev/null) || echo "sudo -E")
+SUDO=$(shell (docker info >/dev/null) || echo "sudo -E")
```

Comment:
Shouldn't that redirect stderr too?
```
$ docker info >/dev/null
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
```
Also the inner parentheses are superfluous, aren't they?